src/parameters/create_parameters.jl

```diff
@@ -7,9 +7,6 @@ import SurfaceFluxes.UniversalFunctions as UF
 import Insolation.Parameters as IP
 import Thermodynamics as TD
 import CloudMicrophysics as CM
-# TODO: Remove these imports?
-import ClimaCore
-import ClimaCore as CC
```

Comment:
What's the purpose of this change with regards to moving parameter logging?

Reply:
They don't seem to serve any purpose, I think they are old fragments that someone forgot to delete.

Reply:
Maybe they should be removed in a different commit if they are not logically related to the change under consideration?